python/paddle/fluid/tests/unittests/dist_se_resnext.py

```diff
@@ -124,7 +124,7 @@ def net(self, input, class_dim=1000):
 
         pool = fluid.layers.pool2d(
             input=conv, pool_size=7, pool_type='avg', global_pooling=True)
-        drop = fluid.layers.dropout(x=pool, dropout_prob=0.2)
+        drop = fluid.layers.dropout(x=pool, dropout_prob=0.2, seed=1)
```

Comment:
`main_program.random_seed` can set the seed of `layers.dropout` now.

https://github.com/PaddlePaddle/Paddle/blob/3300a532aa05ada96fde9fd908b562b3f32da6d9/python/paddle/fluid/layers/nn.py#L953-L954